### PR TITLE
AtBot Admin Connector

### PR DESCRIPTION
**Pull request by MickH3**
Initial submission for AtBot Admin connector.

---
Please check the following conditions for your PR.

- [X ] `apiDefinition.swagger.json` is validated using `paconn validate` command.
- [ X] `apiProperties.json` has a valid brand color. Invalid brand colors are `#007ee5` and `#ffffff`.

### CODE REVIEW

File: certified-connectors/AtBot Admin/apiProperties.json [hunk 1]
```diff
@@ -0,0 +1,34 @@
+{
+  "properties": {
+    "capabilities": [],
+    "connectionParameters": {
+      "token": {
+        "oAuthSettings": {
+          "clientId": "6145b6dc-1ce4-43b1-a4b5-a19b8440096a",
```
Comment (dougelliott) [inline certified-connectors/AtBot Admin/apiProperties.json:7]: This can be removed and replaced with a dummy value

Comment (MickH3) [reply certified-connectors/AtBot Admin/apiProperties.json]: I've removed and replaced with a dummy value.  Thanks!

File: certified-connectors/AtBot Admin/apiProperties.json [hunk 2]
```diff
@@ -4,13 +4,13 @@
     "connectionParameters": {
       "token": {
         "oAuthSettings": {
-          "clientId": "6145b6dc-1ce4-43b1-a4b5-a19b8440096a",
+          "clientId": "CLIENT ID",
```
Comment (joem-msft) [inline certified-connectors/AtBot Admin/apiProperties.json:7]: you'll see a better UI experience (and others that try to work on it) if you just set it to `null`.
`null` is not a valid value at deployment time, and so the error will happen right away, rather become some weird runtime issue.

Same below with the resourceUri

Comment (MickH3) [reply certified-connectors/AtBot Admin/apiProperties.json]: Thanks for the feedback.  I've changed them to null.

**Review by dougelliott — APPROVED** [review]

**Pull request merged**